gm4_blossoming_pots/beet.yaml

```
@@ -0,0 +1,41 @@
+id: gm4_blossoming_pots
+name: Blossoming Pots
+version: 3.1.X
+
+data_pack:
+  load: .
+  overlays:
+    - formats:
+        min_inclusive: 61
+        max_inclusive: 61
+      directory: since_61
+
+resource_pack:
+  load: .
+
+pipeline:
+  - gm4.plugins.extend.module
+
+meta:
+  gm4:
+    versioning:
+      schedule_loops: [main]
+    website:
+      description: Decorated Pots and Flower Pots extended to allow all kinds of things to be placed in them! Decorate in Style!
+      #recommended:
+        #- 
+      notes:
+        - <add notes on version or incompatibilities?>
+    #modrinth: 
+      #project_id: null
+    #smithed:
+      #pack_id: null
+    planetminecraft: 
+      uid: null
+    video: null
+    wiki: null
```

Link to wiki should be added even if the page does not exist yet
```suggestion
    wiki: https://wiki.gm4.co/wiki/Blossoming_Pots
```